CONTRIBUTING.md

```diff
@@ -115,7 +115,7 @@ Start reading our code and you'll get the hang of it. We optimize for readabilit
 * **Curly Braces { }**
     Always use braces even for one line if's. Unity did this everywhere, and there is value in not accidentally missing a line in an if statement because there were no braces.
 * **Variable naming**  
-    \`NetworkIdentity identity\`, not \`NetworkIdentity uv\` or similar. If the variable needs a comment the name needs to be changed. For example, `msg = ... // the message` use `message = ...` without a comment instead
+    Follow [C# standard naming conventions](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md). Also, be descriptive. \`NetworkIdentity identity\`, not \`NetworkIdentity uv\` or similar. If the variable needs a comment the name needs to be changed. For example, `msg = ... // the message` use `message = ...` without a comment instead. Use camelCase, and avoid prefixes like `m_`, `s_`, or similar.
```

Comment:
This contradicts itself.

For example,  if I am writing a class or method,  according to the C# standard I should use PascalCase and according to your comment I should use camelCase

Reply:
Right. The section is on Variable naming, though. Maybe it should be expanded to all naming, not just variable naming, as you say, and remove the bit about camelCase, then?

Reply:
General agreement for most unity users is that unity serialized fields use camelCase @paulpach

Reply:
ok  if this only applies to variable names,  then they are both saying the same thing.

Reply:
I can expand this to be about naming in general